tests/network/sriov.go

```diff
@@ -806,3 +809,46 @@ func trimRawString2JSON(input string) string {
 	}
 	return input[startIdx : endIdx+1]
 }
+
+func getRunningVMIDomainSpec(vmi *v1.VirtualMachineInstance) (*launcherApi.DomainSpec, error) {
+	runningVMISpec := launcherApi.DomainSpec{}
+	cli := kubevirt.Client()
+
+	domXML, err := getRunningVirtualMachineInstanceDomainXML(cli, vmi)
+	if err != nil {
+		return nil, err
+	}
+
+	err = xml.Unmarshal([]byte(domXML), &runningVMISpec)
+	return &runningVMISpec, err
+}
+
+func getRunningVirtualMachineInstanceDomainXML(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (string, error) {
```

Comment:
No need to receive the Go client instance, you can have it by calling `client.Kubeivrt()` directly, its safe to use because its a singleton.